src/hyperlight_host/src/hypervisor/hyperv_linux.rs

```diff
@@ -85,31 +57,17 @@ pub(crate) struct MshvVm {
     vcpu_fd: VcpuFd,
 }
 
-impl HypervLinuxDriver {
-    /// Create a new `HypervLinuxDriver`, complete with all registers
-    /// set up to execute a Hyperlight binary inside a HyperV-powered
-    /// sandbox on Linux.
-    ///
-    /// While registers are set up, they will not have been applied to
-    /// the underlying virtual CPU after this function returns. Call the
-    /// `apply_registers` method to do that, or more likely call
-    /// `initialise` to do it for you.
-    #[allow(clippy::too_many_arguments)]
-    // TODO: refactor this function to take fewer arguments. Add trace_info to rt_cfg
+static MSHV: LazyLock<Result<Mshv>> =
+    LazyLock::new(|| Mshv::new().map_err(|e| new_error!("Failed to open /dev/mshv: {}", e)));
+
+impl MshvVm {
```

Comment:
The `struct MshvVm` was already defined, but the `impl` block had the wrong name. I think it's an artifact of commits editing

Reply:
yeah sorry about that. The final result should still be correct